end_to_end_testing/setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+
+echo "Installing git and jq"
+sudo apt-get install -yqq jq git
+
+echo "Installing Reckoner"
+sudo cp /tmp/binaries/reckoner-linux-amd64 /usr/local/bin/reckoner
+reckoner version
+
+echo "Installing Kind"
+curl -sLO https://github.com/kubernetes-sigs/kind/releases/download/0.2.1/kind-linux-amd64
```

Comment:
Should we update this to 0.4.0 now?  Also, should we pin the Kubernetes version we are using for this?  I think 0.4.0 now defaults to 1.14 or something like that.

Reply:
I will adjust that in a new PR if that's ok - good call out and I will make sure to Pin both the Kind version to 0.4.0 and K8s version to v1.14.x